.github/pull_request_template.md

```diff
@@ -0,0 +1,36 @@
+## PR Title :boom:
+
+Please title this PR with a summary of the change, along with the JIRA card number.
+
+Suggested formats: 
+
+1. Fixes/Refs #RHIROS-XXX - Title
+2. RHIROS-XXX Title 
+
+Feel free to remove this section from PR description once done.
+
+## Why do we need this change? :thought_balloon:
+
+Please include the __context of this change__ here.
+
+## Documentation update? :memo:
+
+- [ ] Yes
+- [ ] No
+
+## Security Checklist :lock:
+
+Upon raising this PR please go through [RedHatInsights/secure-coding-checklist](https://github.com/RedHatInsights/secure-coding-checklist)
+
+## :guardsman: Checklist :dart:
+
+- [ ] Bugfix
+- [ ] New Feature
+- [ ] Refactor
+- [ ] Unittests Added
+- [ ] DRY code
```

Comment:
@saltgen all other checkbox looks good to me but about `DRY` I feel it will be good if the person reviewing the PR observe about the same? thoughts?

Reply:
IMO, DRY practices should be on the contributor, and in any case there are chances of reviewers missing it,
especially if it is not showing up on the diffs/changes